config/profile_ccdl.config

```
@@ -12,6 +12,8 @@ params{
 
   // a set of run_ids for testing
   run_ids = "SCPCR000001,SCPCS000101"
+  // include all runs in a merged project
+  merge_run_ids = "All"
```

I would expand comments a bit, something like - 

```suggestion
  // a set of run_ids for testing. used only by the main workflow
  run_ids = "SCPCR000001,SCPCS000101"
  // include all runs in a merged project. used only by the merged workflow
  merge_run_ids = "All"
```